packages/config/src/functions_config.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const isPlainObj = require('is-plain-obj')
+
+const configProperties = new Set(['externalModules', 'ignoredModules'])
+
+const WILDCARD_ALL = '*'
+
+const isConfigLeaf = (obj) => isPlainObj(obj) && Object.keys(obj).every(isConfigProperty)
+
+const isConfigProperty = (prop) => configProperties.has(prop)
+
+const normalize = (config) => {
```

Comment:
Should we call `config`, `functions` or `functionsConfig` instead, to prevent any confusion with the main `config` object?

Reply:
Done in https://github.com/netlify/build/pull/2390/commits/5f4f3599f50e6ae3c17a0682df95fe9cd7df0e1c.